### PR TITLE
#675 Open ksm transaction and view current price of KSM

### LINKED ISSUE #675: Add current KSM price and link to extrinsic to Transfer dialog
1. Display link for the transaction which was made on Subscan using generated extrinsic, i.e. 
**0x256768b350ed980fd6dfb1d6172807be8765b04f475f3c315f5632bdd0417568** linking to 
https://kusama.subscan.io/extrinsic/0x256768b350ed980fd6dfb1d6172807be8765b04f475f3c315f5632bdd0417568

Somewhere permanent, no in snackbar, best if you can add there copy to clipboard button.

2. Display somewhere current price of $KSM. You can find a way how we fetch prices from coingecko in kodadot.

image

For more check this PR how is Transfer made https://github.com/kodadot/nft-gallery/pull/672

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the KodaDot NFT gallery.
Let's do a quick check before the merge.
Your contribution will be an eternal part of the community codebase

### Before submitting this PR, please make sure:
- [x] Code builds clean without any errors or warnings
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I didn't break any original functionality

### Optional
- [ ] I've tested it on mobile and everything works

### PR type
- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new? (maybe part of changelog)
- closes #675 -  For opening ksm transactions, I have added a button and have found the transaction id using `blockHash.toHex();`
For showing current price of KSM, have used state variable `fiatPrice`. 

### If issue contains bounty label ($,$$,$$$)
- Your KSM address: EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU

### Community participation
- [x] Are you at KodaDot Discord? If no, [let's join us - https://discord.gg/35hzy2dXXh](https://discord.gg/35hzy2dXXh)

### Screenshot
- [x] Whenever your fix will change **something** on UI, a screenshot is more than welcome

Screenshot 2021-09-03 at 9 16 46 PM

### CODE REVIEW

File: src/components/transfer/Transfer.vue [hunk 1]
```diff
@@ -5,12 +5,25 @@
...
         <Loader v-model="isLoading" :status="status" />
         <div class="box">
-          <p class="title is-size-3">
+          <div class="info">
+            <p class="title is-size-3">
             Transfer {{ unit }}
-          </p>
+            </p>
+            <span class="info--currentPrice" title='Current price'>${{this.$store.state.fiatPrice[this.coinId]['usd']}} </span>
+          </div>
```
Comment (vikiival) [inline src/components/transfer/Transfer.vue:13]: You should write a [Vuex getter](https://vuex.vuejs.org/guide/getters.html) for this

File: src/components/transfer/Transfer.vue [hunk 2]
```diff
@@ -165,6 +181,13 @@ export default class Transfer extends Mixins(
     this.isLoading = false;
   }
 
+    protected getExplorerUrl() {
+    const url =  urlBuilderTransaction(this.transactionValue,
+      this.$store.state.explorer.chain,
```
Comment (vikiival) [inline src/components/transfer/Transfer.vue:186]: also getter for this

File: src/components/transfer/Transfer.vue [hunk 3]
```diff
@@ -165,6 +181,13 @@ export default class Transfer extends Mixins(
     this.isLoading = false;
   }
 
+    protected getExplorerUrl() {
+    const url =  urlBuilderTransaction(this.transactionValue,
```
Comment (vikiival) [inline src/components/transfer/Transfer.vue:185]: Use the linter please

**Review by vikiival — CHANGES_REQUESTED** [review]
Otherwise good.
But I would move the button under the form

**Comment by yangwao** [comment]
Why is a field for an address now short?
Balance seems needs spacing from amount 
image
On the production looks like longer, it's better when one wants to see whole address https://nft.kodadot.xyz/transfer

**Comment by prachi00** [comment]
> have done the mentioned changes and updated the screenshot in the PR description

**Comment by yangwao** [comment]
I would show the button only if there was made some transaction.
Now it shows by default and leading to https://kusama.subscan.io/extrinsic/
image

I'm thinking that showing on button first 6 chars from extrinsic would be perfect to identify transaction before clicking

**Comment by prachi00** [comment]
@yangwao 
Screenshot 2021-09-06 at 12 07 14 AM

Have fixed these. Please check now.

**Comment by yangwao** [comment]
@prachi00 you are officially the first one I've used it to pay out the bounty, feels amazing! 

image

**Pull request merged**